### ISSUE TITLE
Proper release of json-schema-ref-parser

### ISSUE THREAD

**Issue by lzgrzebski** [bcherny/json-schema-to-typescript#469]
> @apidevtools/json-schema-ref-parser@https://github.com/bcherny/json-schema-ref-parser.git#commit=984282d34a2993e5243aa35100fe32a63699164d: The remote archive doesn't match the expected checksum

Should we fork and release the version with proper version + tag instead of pointing to arbitrary refs?

**Comment by Kiran554** [comment]
This is not idea. As it creates reference to github which might not be reachable for internal repositories. Please release another version to fix this.

**Comment by bradenneufeld** [comment]
+1, This is breaking our CodeBuild pipeline.

```
[Container] 2022/08/03 05:01:18 Running command npm ci
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/bcherny/json-schema-ref-parser.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '192.30.255.113' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128
```

**Comment by Quentigus** [comment]
+1, I can't use recent releases because of this reference :(

**Comment by bradenneufeld** [comment]
After reviewing https://github.com/npm/cli/issues/2610, I was able to resolve in our CI/CD pipeline (AWS Codebuild) by updating our projects to use the new `aws/codebuild/standard:6.0` image (node 16/npm 8) instead of `aws/codebuild/standard:5.0` (node 14/npm 6).

**Comment by bcherny** [comment]
Forking json-schema-ref-parser for now. Will switch back to the upstream once https://github.com/APIDevTools/json-schema-ref-parser/pull/272 is merged.

Published in 11.0.2.
